dolphin/drivers/fake_storage/__init__.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dolphin.drivers import driver
+
+
+class FakeStorageDriver(driver.StorageDriver):
+    """FakeStorageDriver shows how to implement the StorageDriver,
+    it also plays a role as faker to fake data for being tested by clients"""
+
+    @staticmethod
+    def get_storage_registry():
+        pass
+
+    def register_storage(self, context, register_info):
+        pass
+
+    def get_storage(self, context):
+        pass
+
+    def list_pools(self, context):
+        pass
+
+    def list_volumes(self, context):
+        pass
+
+    def parse_alert(self, context, alert):
+        pass
+
+    def clear_alert(self, context, alert):
```

Comment:
we will be having one more API to config trap at device side

Reply:
yes, can you give me a suggestion to define define them?

Reply:
such as:?
add_snmp_trap(self, context, snmp_trap):
remove_snmp_trap(self, context, snmp_trap):

Reply:
suggest add/remove_trap_config(self, context,trap_config)

Reply:
thank you, done.